src/Tribe/Template.php

```diff
@@ -627,7 +648,7 @@ public function template( $name, $context = array(), $echo = true ) {
 
 		// Only do this if really needed (by default it wont).
 		if ( true === $this->template_context_extract && ! empty( $this->context ) ) {
-			// We don't allow Extrating of a variable called $name
+			// We don't allow Extracting of a variable called $name
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 64a89149819aebf6e7d10738cde0cc7d</sub>